File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    findMin(nums) {
        let min = Number.MAX_VALUE;
        for(let i of nums) {
            if(i < min){
                min = i
            }
        }   
        return min;
    }
}

File: Data Structures & Algorithms/minimum-size-subarray-sum/submission-1.java
class Solution {
    public int minSubArrayLen(int target, int[] nums) {
        int min = Integer.MAX_VALUE;

        for(int i=0;i <nums.length;i++) {
            int sum = 0;
            for(int j=i;j <nums.length;j++) {
                sum += nums[j];

                if(sum >= target) {
                    int len = j - i+ 1;
                    min = Math.min(min, len);
                }
            }
        }
        return min == Integer.MAX_VALUE ? 0 : min;
    }
}